ai-ml/nl-to-sql/manifests/03-llm/tgi.yaml

```diff
@@ -0,0 +1,90 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_ai_ml_nl_to_sql_manifests_03_llm]
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tgi
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: tgi-runtime
+  template:
+    metadata:
+      labels:
+        app: tgi-runtime
+        ai.gke.io/model: llama-3-8b
+        ai.gke.io/inference-server: text-generation-inference
+        examples.ai.gke.io/source: user-guide
+    spec:
+      containers:
+      - name: inference-server
+        image: ghcr.io/huggingface/text-generation-inference:2.0.2
+        resources:
+          requests:
+            cpu: "2"
+            memory: "7Gi"
+            ephemeral-storage: "40Gi"
+            nvidia.com/gpu: 1
+          limits:
+            cpu: "2"
+            memory: "7Gi"
+            ephemeral-storage: "40Gi"
+            nvidia.com/gpu: 1
+        args:
+        - --model-id=$(MODEL_ID)
+        - --num-shard=1
+        env:
+        - name: MODEL_ID
+          #value: google/gemma-2b-it
+          value: dganochenko/llama-3-8b-chat
```

Comment:
Is this intentional?

Reply:
Friendly ping~

Reply:
Hi, i'm not sure should we change it or not, because no reviewer replied about this moment. It's my public repo for models/datasets. The only i thing i can do is removing commented line, will do it a bit later

Reply:
We just found a way how to fix it, will ping you when it's ready to review

Reply:
done